tools/diagnostic-information

```diff
@@ -97,6 +97,30 @@ image_version(){
     fi
 }
 
+licensing_information(){
+    verbose "licensing information..."
+    title 1 "licensing information"
+
+    # Coreutils timeout
+    TIMEOUT=""
+    if which timeout 2>/dev/null >/dev/null ; then
+        TIMEOUT="timeout 5s "
+    fi
+
+    # Notes:
+    # * bmsg request does not return 0 for licensing INFO, since the MB does
+    #   not return "OK", so trap it using " || :" to not have the script aborting
+    # * also delete everything before the first actual information
+    $TIMEOUT bmsg -t 5 request etn-licensing GET REQUEST 1234 INFO 2>/dev/null | sed -n '/overall-state/,$p' 1>/tmp/licensing_info || :
+
+    if [ -f /tmp/licensing_info ] ; then
```

Comment:
If we expect to not get a response (the true-trap above), file would be empty. Better test `-s` than `-f`.

Reply:
true, thx for catching. fixed